src/config.c

```diff
@@ -848,4 +850,27 @@ void loadParodusCfg(ParodusCfg * config,ParodusCfg *cfg)
     }
 }
 
+#ifdef WAN_FAILOVER_SUPPORTED
+void setWebpaInterface(char *value)
+{
+	pthread_mutex_lock (&config_mut);
+	parStrncpy(get_parodus_cfg()->webpa_interface_used, value, sizeof(get_parodus_cfg()->webpa_interface_used));
+	pthread_mutex_unlock (&config_mut);
+}
+#endif
+
+char *getWebpaInterface(void)
+{
+	#ifdef WAN_FAILOVER_SUPPORTED	
+		ParodusInfo("WAN_FAILOVER_SUPPORTED mode \n");
+		pthread_mutex_lock (&config_mut);	
+		parStrncpy(webpa_interface, get_parodus_cfg()->webpa_interface_used, sizeof(webpa_interface));
```

Comment:
Both if and else has same mechanism to fetch interface name from paroduscfg , getWebpaInterface() for wan fail over enabled/disabled cases looks same.
So should we need this if else block , can it not be generic ?

Reply:
Confirmed with shilpa, For normal case since we get the value form property file, lock is not needed there. 
CC : @shilpa24balaji

Reply:
ok